docs/plot/plotdefs.json

```diff
@@ -1053,11 +1053,22 @@
     {
         "filename": "peirce_q.png",
         "latmax": 90,
-        "latmin": 0,
+        "latmin": -90,
         "lonmax": 180,
         "lonmin": -180,
         "name": "peirce_q",
-        "projstring": "+proj=peirce_q",
+        "projstring": "+proj=peirce_q +lon_0=25",
+        "res": "low",
+        "type": "poly"
+    },
+    {
+        "filename": "peirce_q_d.png",
```

Comment:
note: adding a new projection method makes the CI "Plots" job currently fail because of https://github.com/OSGeo/PROJ/issues/2961 . I'll try to look at solving it

Reply:
Solved per https://github.com/OSGeo/PROJ/pull/2981 (you can rebase on top of latest master)